app/js/app.js

```diff
@@ -8,6 +8,13 @@ conceptDictionary.config(['$routeProvider',
                           templateUrl: 'partials/class-list.html',
                           controller: 'ClassesListCtrl'
                         }).
+                        when('/class-list/addClass', {
+                        	templateUrl: 'partials/class-add.html'                      		
+                        }).
+                        when('/class-list/:className*', {
```

Comment:
should be "`:className`" instead of "`:className*`". Now directs straight to edit view.